spec/spec_helper.rb

```diff
@@ -23,8 +23,6 @@ def rollback_transaction; end
   require file
 end
 
-eager_load!
```

Comment:
the related file in which the implementation for this method lives should be removed too

Reply:
see #2216